spp_import_dci_api/models/fetch_crvs_beneficiary.py

```diff
@@ -0,0 +1,1015 @@
+import copy
+import json
+import logging
+import uuid
+from datetime import datetime, timezone
+from urllib.parse import urlencode
+
+import requests
+from dateutil import parser
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import safe_eval
+
+from ..models import constants
+
+_logger = logging.getLogger(__name__)
+
+
+class FetchDomainFilter(models.TransientModel):
+    _name = "spp.fetch.domain.filter"
+    _description = "Fetch Domain Filter"
+
+    birthdate = fields.Date("Birth Date")
+
+
+class CRVSLocationType(models.Model):
+    _name = "spp.crvs.location.type"
+    _description = "CRVS Location Type"
+
+    name = fields.Char(compute="_compute_name")
```

Comment:
```suggestion
    name = fields.Char(compute="_compute_name", store=True)
```
I don't think location type gonna change a lot, or instead of defining new field, you can just define your own `name_get()` function to return what you want